### DISCUSSION TITLE
Zeek not saving extracted files

### DISCUSSION THREAD

**Discussion by Yawndy** [Security-Onion-Solutions/securityonion#3092] [Unsupported Versions]
Hello,

I'm currently using SO 2.3.1 and I noticed when I import a PCAP file using sudo-import pcapfilename, the "/zeek/extracted folder" contains a directory "completed" but it's empty. I've left all configurations to the default settings, but I can see the Zeek extraction alerts in the Hunt section from SO. I was wondering if I'm doing something wrong since I can see the alert that indicates the filename extracted.

Any help is greatly appreciated!

**Comment by weslambert** [comment]
Are you running Strelka?  If so, try checking the `/nsm/strelka/processed` folder.

**Reply by Yawndy** [reply]
Hello,

I am running Strelka, but I checked /nsm/strelka/processed, and its still empty unfortunately.
